configs/detection/ava/README.md

```diff
@@ -80,6 +80,18 @@ python tools/train.py configs/detection/AVA/slowonly_kinetics_pretrained_r50_8x8
 
 For more details and optional arguments infos, you can refer to **Training setting** part in [getting_started](/docs/getting_started.md#training-setting) .
 
+### Train Custom Classes From Ava Dataset
```

Comment:
Can also add a trained ckpt, introduce the classes you use (better have some reasons), and add the performance.